public/js/components/SourceTabs.js

```diff
@@ -139,6 +140,13 @@ const SourceTabs = React.createClass({
       CloseButton({ handleClick: onClickClose }));
   },
 
+  renderNewButton() {
+    return dom.div({
+      className: "new-tab-btn",
+      onClick: () => this.props.openFileSearch(true)
```

Comment:
ahh - good point. I guess the next best thing to do would be to make the button a no-op when file search is open.